src/runtime/database.cpp

```diff
@@ -1584,6 +1589,25 @@ std::vector<JobTag> Database::get_tags() {
   return out;
 }
 
+static std::vector<RunReflection> get_all_runs(const Database *db, sqlite3_stmt *query) {
```

Comment:
... but here you're using that pattern already ...

(Side note, this seems to be the only implementation here which uses a separate `static` function rather than having it all in the `Database::` method.)

Reply:
Sounds good, yeah removed the static function ( I was just following the get_file_dependencies pattern: https://github.com/sifive/wake/blob/4d62b1d4a50d7e6ff99c32abbc6d8a3999669dce/src/runtime/database.cpp#L1626 , but realized that is unnecessary for get_runs)